Reference/Querying/UmbracoHelper/index.md

```diff
@@ -33,16 +24,17 @@ The samples below demonstrate using `UmbracoHelper` in Razor. Aside from the `@`
 
 ## Working with Content
 
-### .Content(int id)
+### .Content(Guid id)
 
-Given a node ID, returns a `dynamic` object, representing a single `IPublishedContent` entity
+Given a node ID , returns a `IPublishedContent`
 
 ```csharp
 @{
-    var page = Umbraco.Content(1234);
+    var page = @Umbraco.Content(Guid.Parse("ca4249ed-2b23-4337-b522-63cabe5587d1"));
 }
 
-<h3>@page.PropertyAlias</h3>
+<h3>@page.Value<string>("propertyAlias") 
```

Comment:
For some reason, I can't get this working when I use `<string>` - if I remove that part, it works fine though 🤔

Any thoughts?

Reply:
Could be a syntax error, I have updated it to `@(page.Value<string>("propertyAlias"))`